build/compile/Microsoft.DotNet.Cli.LzmaArchive.targets

```diff
@@ -80,12 +80,12 @@
 
       <Message Text="Publishing Archiver" />
 
-      <DotNetPublish ToolPath="$(Stage0Directory)"
+      <DotNetPublish ToolPath="$(Stage2Directory)"
```

Comment:
Again, this change is needed.  Archiver fails because it targets 1.0.3 now, and Stage0 only installs 1.0.1.